omnibus/config/projects/updater.rb

```diff
@@ -17,6 +17,17 @@
 
 install_dir INSTALL_DIR
 
+if ENV.has_key?("OMNIBUS_WORKERS_OVERRIDE")
+  COMPRESSION_THREADS = ENV["OMNIBUS_WORKERS_OVERRIDE"].to_i
+else
+  COMPRESSION_THREADS = 1
+end
+if ENV.has_key?("DEPLOY_AGENT") && ENV["DEPLOY_AGENT"] == "true"
+  COMPRESSION_LEVEL = 9
+else
+  COMPRESSION_LEVEL = 5
+end
```

Comment:
❓ Question :
Is it so that dev builds are faster than release ones ?

Reply:
Yes, this is what we do for other projects.
The basic reasonning is that for deploy builds (ideally this should be based on a "is_release" flag but I don't think we have such a flag) we will have to pay for the bandwidth when users download the product, so we're fine spending a bit of extra time compressing it.
For internal builds, we don't really care about bandwidth and mostly want to have a build ready fast